internal/app/pactproxy/constraint.go

```diff
@@ -15,3 +18,34 @@ type interactionConstraint struct {
 func (i interactionConstraint) Key() string {
 	return strings.Join([]string{i.Interaction, i.Path}, "_")
 }
+
+func (i interactionConstraint) Check(expectedValues []interface{}, actualValue interface{}) error {
```

Comment:
Does it need to be exported? :thinking:
```suggestion
func (i interactionConstraint) check(expectedValues []interface{}, actualValue interface{}) error {
```